### PR TITLE
Remove the MorrisExperiment interface

### PR DESCRIPTION
**Pull request by jschueller**
This explicits the API: it's either a MorrisExperimentGrid or a MorrisExperimentLHS.
Also simplified the API of Morris with a single ctor with a WeightedExperiment instead of the combination of all the MorrisExperiments ctors. We simply check for the right size N*(p+1).
Also added examples for MorrisExperimentLHS/MorrisExperimentGrid in docstring. Mentioned that we need centered lhs.

### CODE REVIEW

File: lib/src/Morris.cxx [hunk 1]
```diff
@@ -41,137 +41,57 @@ Morris::Morris(const Sample & inputSample, const Sample & outputSample)
...
   if (size != N * (inputDimension + 1))
     throw InvalidArgumentException(HERE) << "In Morris::Morris, sample size should be a multiple of " << inputDimension + 1;
   // Perform evaluation of elementary effects
   computeEffects(N);
 }
 
 /** Standard constructor with levels definition, number of trajectories, model */
-Morris::Morris(const Indices & levels, const UnsignedInteger N, const Function & model)
+Morris::Morris(const WeightedExperiment & experiment, const Function & model)
   : PersistentObject()
```
Comment (sofianehaddad) [inline lib/src/Morris.cxx]: You mean MorrisExperiment?

File: python/src/Morris_doc.i.in [hunk 2]
```diff
@@ -5,40 +5,19 @@ Available constructors:
...
-N : int
-    Number of trajectories
-
-interval : :py:class:`openturns.Interval`
-    Bounds of the domain
-
-lhsDesign : :py:class:`openturns.Sample`
-    Initial design
-
+experiment : :py:class:`openturns.WeightedExperiment`
```
Comment (sofianehaddad) [inline python/src/Morris_doc.i.in]: experiment : :py:class:`otmorris.MorrisExperiment`

**Comment by jschueller** [comment]
ok

**Comment by sofianehaddad** [comment]
Maybe mention in ChangeLog?

**Comment by jschueller** [comment]
There's no changelog :)

**Comment by jschueller** [comment]
I populated the changelog

**Comment by sofianehaddad** [comment]
Seems ok for me, +1

**Pull request merged**